### ISSUE TITLE
Is there any plan to update CPP g4 to C++17 or C++20?

### ISSUE THREAD

**Issue by lxz12523** [antlr/grammars-v4#3095]

**Comment by kaby76** [comment]
Yes. I have a scraper that takes all versions of the Spec, and outputs an unoptimized grammar. https://github.com/kaby76/scrape-c-plus-plus-spec/tree/main/scraper. Still work in progress.
